tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1,11 +1,15 @@
 import base64
```

Comment:
This is a big chunk of tests but I think it's important to review test cases properly to make sure I didn't miss a possible scenario